examples/stem_histogram.py

```diff
@@ -0,0 +1,91 @@
+import glob
+import sys
+import os
+from stempy import io, image
+import numpy as np
+from PIL import Image
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+def print_help():
+    print(
+    '''
+    Usage:
+    python3 stem_histogram.py [dataDir -- path to the data directory]
+                              [outDir -- path to the output image directory]
+    ''')
+
+def save_img(stem_image_data, name):
+    min = np.min(stem_image_data)
+    max = np.max(stem_image_data)
+
+    stem_image_data = stem_image_data.reshape((160, 160))
+    stem_image_data = np.interp(stem_image_data, [min, max], [0, 256])
+    stem_image_data = stem_image_data.astype(np.uint8)
+    img = Image.fromarray(stem_image_data)
+    img.save(name)
+
+
+def main(argv):
+    print_help()
+    dataDir = argv[0] if len(argv) > 0 else '/home/zhuokai/Desktop/data/data*.dat'
+    outDir = argv[1] if len(argv) > 1 else '/home/zhuokai/Desktop/stempy/examples/output/'
```

Comment:
It's probably okay to leave the argument parsing here as-is for now. But if it gets any more complicated in the future, I would recommend using [click](https://click.palletsprojects.com/en/7.x/) as we have done in some of the other [sample files](https://github.com/OpenChemistry/stempy/blob/1f4eb12cdbf8a845bd9e849e6a6a72d5ad1b09df/examples/create_hdf5.py#L7). It's a nice argument parsing library, and it's already a dependency of stempy.

Reply:
I think it would be best to remove hard-coded paths from our code, keep it locally if you like, but please remove it.

Reply:
@psavery Thanks, I've changed the current code to use click. @cryos Sure, I have removed the paths.